prow/tide/tide.go

```diff
@@ -234,6 +237,45 @@ func accumulate(presubmits []string, prs []pullRequest, pjs []kube.ProwJob) (suc
 	return
 }
 
+func prNumbers(prs []pullRequest) []int {
+	var nums []int
+	for _, pr := range prs {
+		nums = append(nums, int(pr.Number))
+	}
+	return nums
+}
+
+func (c *Controller) pickBatch(sp subpool) ([]pullRequest, error) {
```

Comment:
How are we going to ensure that the single PR which we have triggered tests for is going to be included here?

Reply:
It's not guaranteed to be here currently. The reason why we might want it to be is that it gives it a second chance to dodge a flake, but I'm not sure how important that is. This does mean that a PR might be retested at the same time as the batch, pass tests, but not merge due to the batch taking priority.

If we settle on a testing order then we can ensure that the single PR is in the batch pretty easily. Currently the submit queue uses least-recently-lgtm'd, but we can also go with oldest PR or something.

Reply:
Would be nice to capture into an issue for more discussion.

Reply:
There's some discussion in the design doc, but it's mostly marked as resolved.